.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@mate-academy/eslint-config",
+  "env": {
+    "jest": true
+  },
+  "rules": {
+    "no-proto": 0
```

Comment:
The `no-proto` rule is set to `0`, which means it is turned off. If this is intentional, you can leave it as is. However, if you want to enforce this rule, consider setting it to `1` (warn) or `2` (error).